recipes/gegles-spdlog_setup/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,19 @@
+from conans import ConanFile, CMake
```

Comment:
You no longer need to add test_v1_package folder for new recipes: https://github.com/conan-io/conan-center-index/discussions/17406